views/fe-utils/fetchUtil.js

```diff
@@ -132,16 +138,44 @@ const deleteChart = async (id) => {
   return res.json()
 }
 
-export { 
+const editChartForm = async (formValues) => {
+  const res = await window.fetch(
+    `${routes.basePath}/api/v1/charts/${formValues._id}`,
```

Comment:
same here with moving this path to the routes object

Reply:
https://github.com/AMP-SCZ/dpdash/pull/118/commits/a02b5f9495eed90745bb37c6f90df4df8e963870